src/accelerate/accelerator.py

```diff
@@ -101,12 +106,29 @@ def __init__(
         device_placement: bool = True,
         split_batches: bool = False,
         fp16: bool = None,
+        mixed_precision: str = None,
         cpu: bool = False,
         deepspeed_plugin: DeepSpeedPlugin = None,
         rng_types: Optional[List[Union[str, RNGType]]] = None,
         dispatch_batches: Optional[bool] = None,
         kwargs_handlers: Optional[List[KwargsHandler]] = None,
     ):
+
+        if mixed_precision:
```

Comment:
```suggestion
        if mixed_precision is not None:
```
We usually prefer explicit tests instead of relying on Python's bool conversion.